REST_API/Implementation/settings/settings.py

```diff
@@ -7,19 +6,20 @@
 
 class Settings:
     implementation_dir = Path(__file__).absolute().parent.parent
-    deployment_data = f"{implementation_dir}/model_deployment_data/"
-    offline_storage = f"{implementation_dir}/storage/"
-    offline_blueprints = "{}blueprints/".format(offline_storage)
-    offline_log = "{}log/".format(offline_storage)
-    ssh_keys_location = "/root/.ssh/"
+    deployment_data = implementation_dir / Path('deployment_data')
+    offline_storage = implementation_dir / Path('storage')
+    offline_log = offline_storage / Path('log')
+    ssh_keys_location = Path("/root/.ssh/")
     interpreter = None
+    git_config = None
+    workdir = Path("/tmp/xopera/git_db/")
 
     key_pair = ""
     logfile_name = "logs.log"
 
     verbose = False
 
-    connection = {
+    sql_config = {
         "host": "172.17.0.3",
```

Comment:
Do we need these default IPs here?

Reply:
No, I agree, this is not the prettiest way, commit below fixes the issues